modules/oadp-1-4-1-release-notes.adoc

```diff
@@ -0,0 +1,138 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-1-4-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="oadp-1-4-1-release-notes_{context}"]
+= OADP 1.4.1 release notes
+
+The {oadp-first} 1.4.1 release notes lists new features, resolved issues and bugs, and known issues.
+
+[id="new-features-1-4-1_{context}"]
+== New features
+
+.New DPA fields to update client qps and burst
+
+You can now change Velero Server Kubernetes API queries per second and burst values by using the new Data Protection Application (DPA) fields. The new DPA fields are `spec.configuration.velero.client-qps` and `spec.configuration.velero.client-burst`, which both default to 100.
+link:https://issues.redhat.com/browse/OADP-4076[OADP-4076]
+
+.Enabling non-default algorithms with Kopia
+
+With this update, you can now configure the hash, encryption, and splitter algorithms in Kopia to select non-default options to optimize performance for different backup workloads.
+
+To configure these algorithms, set the `env` variable of a `velero` pod in the `podConfig` section of the DataProtectionApplication (DPA) configuration. If this variable is not set, or an unsupported algorithm is chosen, Kopia will default to its standard algorithms.
+link:https://issues.redhat.com/browse/OADP-4640[OADP-4640]
+
+
+[id="resolved-issues-1-4-1_{context}"]
+== Resolved issues
+
+.Restoring a backup without pods is now successful
+
+Previously, restoring a backup without pods and having `StorageClass VolumeBindingMode` set as `WaitForFirstConsumer`, resulted in the `PartiallyFailed` status with an error: `fail to patch dynamic PV, err: context deadline exceeded`. 
+With this update, patching dynamic PV is skipped and restoring a backup is successful without any `PartiallyFailed` status.
+link:https://issues.redhat.com/browse/OADP-4231[OADP-4231]
+
+
+.PodVolumeBackup CR now displays correct message
+
+Previously, the `PodVolumeBackup` custom resource (CR) generated an incorrect message, which was: `get a podvolumebackup with status "InProgress" during the server starting, mark it as "Failed"`.
+With this update, the message produced is now:
+----
```

Comment:
```suggestion
[source,text]
----
```

Reply:
You need this metadata per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#system-messages